engine/apps/mobile_app/tasks.py

```diff
@@ -87,8 +139,18 @@ def _create_error_log_record():
 
     alert_body = f"Status: {status_verbose}, alerts: {alerts_count_str}"
 
+    mobile_app_user_settings, _ = MobileAppUserSettings.objects.get_or_create(user=user)
+
+    # APNS only allows to specify volume for critical notifications
+    apns_volume = mobile_app_user_settings.critical_notification_volume if critical else None
```

Comment:
The volume setting here is passed to APNS which only applies to iOS, but `default_notification_volume` is only used by Android. I wanted to explicitly pass `None` here instead of `default_notification_volume` so it's clear that APNS doesn't take that parameter into account.